incubator/oauth-proxy/README.md

```diff
@@ -45,6 +45,7 @@ Parameter | Description | Default
 `config.clientID` | oauth client ID | `""`
 `config.clientSecret` | oauth client secret | `""`
 `config.cookieSecret` | server specific cookie for the secret; create a new one with `python -c 'import os,base64; print base64.b64encode(os.urandom(16))'` | `""`
+`config.configFile` | custom [oauth_proxy.cfg](https://github.com/bitly/oauth2_proxy/blob/master/contrib/oauth2_proxy.cfg.example) contents for settings not overridable via environment nor command line | `""`
```

Comment:
Can we change this to `custom [oauth2_proxy.cfg]` just for consistency within the chart?

Reply:
I've taken care of this in PR: https://github.com/kubernetes/charts/pull/4744